src/screens/Home/Home.js

```diff
@@ -331,6 +343,14 @@ class HomeScreen extends React.Component<Props, State> {
     });
   };
 
+  goToProfileEmailSettings = () => {
+    const { navigation } = this.props;
+    this.setState({ forceCloseLoginApprovalModal: true }, () => {
```

Comment:
Could you explain why you 're doing this trick?
And it would be good to put this answer into a comment in the code. Use `// HACK: ...` or `// NOTE: ...`